src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import './Navbar.css'
+
+export const Navbar = () => {
+  type StyleProp = {
+    isActive: boolean
+  }
+  type Style = {
+    color: string
+    textDecoration: string
+  }
```

Comment:
type should be declared in a separate type file
You don't need to create a Style type if we switch to class base approach

Reply:
Removed types from navbar.